README.md

```diff
@@ -674,4 +679,4 @@ Using the index object:
 
 <hr>
 
-**Meilisearch** provides and maintains many **SDKs and Integration tools** like this one. We want to provide everyone with an **amazing search experience for any kind of project**. If you want to contribute, make suggestions, or just know what's going on right now, visit us in the [integration-guides](https://github.com/meilisearch/integration-guides) repository.
+Meilisearch provides and maintains many SDKs and Integration tools like this one. We want to provide everyone with an **amazing search experience for any kind of project**. For a full overview of everything we create and maintain, take a look at the [integration-guides](https://github.com/meilisearch/integration-guides) repository.
```

Comment:
Should we use `Integration` or just `integration`?

Reply:
should we make it plural?

Reply:
You mean in "Meilisearch provides and maintains many SDKs and Integration tools […]", correct?

I'm 100% with writing "integration tools" in lowercase. We shouldn't make integrations plural, though—in this case, "integration" works as an adjective for "tools", so plural would be wrong (like "fats cats").

Reply:
I agree !